### ISSUE TITLE
Incorporate out of system contributions and Additional Funding Panel

### ISSUE THREAD

**Issue by zachwhelchel** [churchof/churchof_rails#155]
This will be tricky... we want churches to be able to add checks made out to ROSM to the system so older generations can still be a part. This will most likely be a Panel where someone is able to make special contributions that are listed with a check number or something on them... It will list who added it, where, when, and "documentation" like the check number... at first this will only be possible though this panel and this role but later will most likely be added to admins and leaders... and checks won't need to flow through ROSM... but right now since there is no clear way for Admins to see on the background how much should be coming to them from the payout and how much was already given through cash... we'll just stick with the special Role and all things going straight to ROSM then to go back out.

Bottom line is this enables in-church kiosk giving or bulletin insert giving, etc.
